skimage/feature/util.py

```diff
@@ -96,18 +101,24 @@ def plot_matches(ax, image1, image2, keypoints1, keypoints2, matches,
         new_image2[:image2.shape[0], :image2.shape[1]] = image2
         image2 = new_image2
 
-    image = np.concatenate([image1, image2], axis=1)
-
-    offset = image1.shape
+    offset = np.array(image1.shape)
+    if alignment == 'horizontal':
+        image = np.concatenate([image1, image2], axis=1)
+        offset[0] = 0
+    elif alignment == 'vertical':
+        image = np.concatenate([image1, image2], axis=0)
+        offset[1] = 0
+    else:
+        raise ValueError('Incorrect value for alignment: %s' % alignment)
```

Comment:
Use `.format` instead

Reply:
How about we also use our New and Improved Error Messages™️ with extra info:

```
mesg = ("plot_matches accepts either 'horizontal' or 'vertical' for "
        "alignment, but '{}' was given. See http://scikit-image.org/"
        "docs/dev/api/skimage.feature.html#skimage.feature.plot_matches "
        "for details.").format(alignment)
raise ValueError(mesg)
```
(you might need to fiddle with the string length, but you get my gist.)